hack/verify.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
```

Comment:
Nit for followup: I think I would rather use REPO_ROOT or something not so tied to k/k script conventions. Repo root is what I saw in some test-infra scripts. 

I don't feel strongly about this but it would be good to standardize across the scripts we have in this repo. This should at least use `BASH_SOURCE[0]` as a followup